source/coding-the-legislation/40_legislation_evolutions.md

```diff
@@ -154,3 +154,100 @@ Note that:
 - The `end` day is **inclusive**: it is the last day a variable and its formulas are active (and not the first day it is not active anymore).
 - The `end` value is a string of format `YYYY-MM-DD` where `YYYY`, `MM` and `DD` are respectively a year, month and day.
 - When defining a date, the month is given **before** the day.
+
+
+## Ending a parameter at a specific date
+
+Similarly to variables, parameters are prone to disappear as the legislation evolves.
+
+For instance, let's assume that the `housing_allowance` is only defined from the 1st of January 2010 to the 30th of November 2016.
+
+We can end `housing_allowance` at a specific date by simply entering **null** as a value for that date (`2016-12-01` in this case).
+
+```yaml
+description: Housing allowance amount (as a fraction of the rent)
```

Comment:
In which file is that?

```suggestion
```yaml
# in parameters/housing_allowance.yaml
description: Housing allowance amount (as a fraction of the rent)
```

Reply:
I added the file reference. However since I changed the end date to the 31/12/2016 in the the doc, the example and the file are no longer consistent.

Reply:
> the example and the file are no longer consistent

I'm not sure I understand.

I don't think it is very important that the file matches a real one somewhere; this is more to explain where the `housing_allowance` parameter name comes from 🙂